<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWindow</class>
 <widget class="QWidget" name="ServerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>服务器</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLineEdit" name="datagram"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="writeButton">
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="stateButton">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>打开服务器</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="buildButton">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>新建客户端</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="output">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>服务器信息</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="serverAddressLabel">
        <property name="text">
         <string>地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="serverAddress">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="serverPortLabel">
        <property name="text">
         <string>端口：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="serverPort">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>serverAddress</tabstop>
  <tabstop>serverPort</tabstop>
  <tabstop>output</tabstop>
  <tabstop>datagram</tabstop>
  <tabstop>writeButton</tabstop>
  <tabstop>stateButton</tabstop>
  <tabstop>buildButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
